### ISSUE TITLE
Does browser cache downloaded images so it does not download them second time around?

### ISSUE THREAD

**Issue by msimonetta** [ideaismobile/IDMPhotoBrowser#173]
I'm presenting the photo browser modally and passing in photos that need to be downloaded. What I am wondering is, if after dismissing the photo browser and re-opening it, will the images that were already downloaded be cached and not downloaded again?

**Comment by maheshm504** [comment]
yes it will not download because SDWebimage will take care of this once the image is downloaded as cache and next time if the same image URL comes first this will check the image Cache is available or not. If available it will not download and show the image .if not it downloads
